src/nrniv/CMakeLists.txt

```diff
@@ -354,3 +354,23 @@ install(TARGETS nrniv_lib DESTINATION ${NRN_INSTALL_SHARE_LIB_DIR})
 if (LIBIVX11DYNAM_NAME)
   install(FILES ${PROJECT_BINARY_DIR}/lib/${LIBIVX11DYNAM_NAME} DESTINATION lib)
 endif()
+
+# =============================================================================
+# Install / copy cpp and required headers for binary special flavor
+# =============================================================================
+# ~~~
+# Note: we also added following headers in NeuronFleLists.cmake:
+#   ivstream.h nrnconf.h nrnmpi.h nrnmpidec.h
+# ~~~
```

Comment:
`nitpick` : this is not needed to keep?

Reply:
Indeed not needed, just wanted to point out what we did. Will remove.